src/ophyd_async/panda/panda_utils.py

```diff
@@ -0,0 +1,22 @@
+from typing import List, Optional
+
+from ophyd_async.core import get_signal_values, save_to_yaml, walk_rw_signals
+from ophyd_async.panda import PandA
+
+
+def _get_panda_phases(panda: PandA, ignore: Optional[List[str]] = None):
+    # Panda has two load phases. If the signal name ends in the string "UNITS",
+    # it needs to be loaded first so put in first phase
+    signals = walk_rw_signals(panda)
+    phase_2 = yield from get_signal_values(signals, ignore=ignore)
+    phase_1 = {n: phase_2.pop(n) for n in list(phase_2) if n.endswith("units")}
+    return [phase_1, phase_2]
```

Comment:
Do we anticipate there will be other devices, for which we'd probably want to split units out like this?

Reply:
As far as I'm aware this logic was specific to the Panda but @coretl might be aware of future devices which will do the same thing

Reply:
There are likely to be more in the future as we add more PVs (e.g. remember the trouble we had with setting TriggerMode at the same time as TriggerSource in the GigE cameras)